recipe/run_conda_forge_build_setup_win.bat

```diff
@@ -24,8 +25,9 @@ if "%CONDA_BLD_PATH%" == "" (
 :: Increase pagefile size, cf. https://github.com/conda-forge/conda-forge-ci-setup-feedstock/issues/155
 if "%CI%" == "azure" (
     if "%CONDA_BLD_PATH%" == "C:\\bld\\" (
```

Comment:
This condition is not true for numpy. You need to add another condition to check `D:\\bld\` and set the pagefile to be in `C:`

Reply:
Thanks, done!